components/rmrk/Gallery/Item/Price.vue

```diff
@@ -0,0 +1,24 @@
+<template>
```

Comment:
move this component to shared folder?

Reply:
We need a bit better folder stucture imo https://github.com/kodadot/nft-gallery/discussions/3068